HaikuPorter/BuildMaster.py

```diff
@@ -843,14 +212,15 @@ def __init__(self, portsTreePath, packagesPath, options):
 
 				builder = None
 				try:
-					builder = RemoteBuilder(configFilePath, packagesPath,
```

Comment:
eventually, i'm going to set a "builder type" field to the builder config and choose from the supported builder types (local, ssh, digitalocean, vultr, etc)